packages/client/scripts/postinstall.js

```diff
@@ -197,30 +194,49 @@ function run(cmd, params, cwd = process.cwd()) {
   })
 }
 
-async function ensureEmptyDotPrisma() {
+/**
+ * Copies our default "throw" files into the default generation folder. These
+ * files are dummy and informative because they just throw an error to let the
+ * user know that they have forgotten to run `prisma generate`.
+ */
+async function createDefaultGeneratedThrowFiles() {
   try {
     const dotPrismaClientDir = path.join(__dirname, '../../../.prisma/client')
+    const defaultNodeIndexPath = path.join(dotPrismaClientDir, 'index.js')
+    const defaultBrowserIndexPath = path.join(dotPrismaClientDir, 'index-browser.js')
+    const defaultNodeIndexDtsPath = path.join(dotPrismaClientDir, 'index.d.ts')
```

Comment:
```suggestion
    const defaultNodeIndexDtsPath = path.join(dotPrismaClientDir, 'index.d.ts')
    const defaultBrowserIndexPath = path.join(dotPrismaClientDir, 'index-browser.js')
```
switch lines two "NodeIndex...Path" are next to each other.